db/migrate/20180904122343_create_hostgroup_content_facet.katello.rb

```diff
@@ -0,0 +1,16 @@
+class CreateHostgroupContentFacet < ActiveRecord::Migration[5.1]
```

Comment:
Interesting date on this migration file ... `20180904122343_create_hostgroup_content_facet.katello.rb` :laughing: